website/docs/plugin/sdkv2/resources/write-only-attributes.mdx

```diff
@@ -0,0 +1,177 @@
+    - The one exception to this case is if the write-only attribute is added to `requires_replace` via [CustomizeDiff](/terraform/plugin/sdkv2/resources/customizing-differences), in that case, the write-only attribute will always cause a diff/trigger a resource recreation
+
+- Since write-only attributes can accept ephemeral values, write-only attribute configuration values are not expected to be consistent between plan and apply.
+
+## Schema Behavior
+
+**Schema example:**
+
+```go
+"password_wo": {
+  Type:     schema.TypeString,
+  Required:  true,
+  WriteOnly: true,
+},
+```
+
+**Restrictions:**
+
+- Cannot be used in data source or provider schemas.
+- Must be set with either `Required` is `true` or `Optional` is `true`
+- Cannot be used when `Computed` is `true`
+- Cannot be used with aggregate schema types (e.g. `typeMap`, `typeList`, `typeSet`)
+
+## Retrieving Write-only Values
+
+Write-only attribute values are only available in the raw resource configuration, you cannot retrieve it using `(*resourceData).Get()` like other attribute values.
+
+Use the `(*schema.ResourceData).GetRawConfigAt()` method to retrieve the raw config value.
+
+This method is an advanced method that uses the `hashicorp/go-cty` library for its type system.
+
+```go
+woVal, diags := d.GetRawConfigAt(cty.GetAttrPath("password_wo"))
+```
+
+### cty.Path
+
+`(*schema.ResourceData).GetRawConfigAt()` uses `cty.Path` to specify locations in the raw configuration.
+
+This is very similar to the `terraform-plugin-framework` [paths](/terraform/plugin/framework/handling-data/paths) or `terraform-plugin-testing` [json paths](/terraform/plugin/testing/acceptance-tests/tfjson-paths).
+
+All top level attributes or blocks can be referred to using `cty.GetAttrPath()`
+
+```go
+cty.GetAttrPath("top_level_schema_attribute")
+```
+
+Maps or map nested blocks can be traversed using `IndexString()`
+
+```go
+// Map traversal
+cty.GetAttrPath("map_attribute").IndexString("key1")
+
+
+// Map nested block traversal
+cty.GetAttrPath("map_nested_block").IndexString("block1").getAttr("map_nested_block_attribute")
+
+```
+
+Lists or list nested blocks can be traversed using `IndexInt()`
+
+```go
+// List traversal
+cty.GetAttrPath("list_attribute").IndexInt(0)
+
+
+// List nested block traversal
+cty.GetAttrPath("list_nested_block").IndexInt(1).getAttr("list_nested_block_attribute")
+```
+
+Sets or set nested blocks can be traversed using `Index()`. `Index()` takes in a `cty.Value` of the set element that you want to traverse into.
+
+```go
+// Set traversal
+cty.GetAttrPath("set_attribute").Index(cty.String("set_string_val"))
+
+
+// Set nested block traversal
+cty.GetAttrPath("set_nested_block").
+Index(cty.ObjectVal(map[string]cty.Value{"set_nested_block_attribute": cty.StringVal("valueA")}))
+.GetAttr("set_nested_block_attribute") // returns path to cty.String("valueA")
+
+```
```

Comment:
This documentation is great! One potential consideration is maybe linking each `cty.Path` example to a Terraform config example. That might help make readers make the mental jump